File: plugins/kotlin/j2k/shared/tests/testData/newJ2k/enum/enumValueOf.java
import static java.lang.Enum.valueOf;

public class Foo {
    public static void main(String[] args) {
        Enum.valueOf(Color.class, "RED");
        foo(Enum.valueOf(Color.class, "RED"));
        valueOf(Color.class, "RED");
        foo(valueOf(Color.class, "RED"));
    }

    private static void foo(Color c) {
    }

    enum Color { RED }
}

File: plugins/kotlin/j2k/new/tests/testData/newJ2k/enum/enumValueOf.kt
object Foo {
    @JvmStatic
    fun main(args: Array<String>) {
        enumValueOf<Color>("RED")
        foo(enumValueOf("RED"))
        enumValueOf<Color>("RED")
        foo(enumValueOf("RED"))
    }

    private fun foo(c: Color) {}
    internal enum class Color {
        RED
    }
}
